src/app_service/common/network_constants.nim

```diff
@@ -271,3 +271,13 @@ var NODE_CONFIG* = %* {
   },
   "OutputMessageCSVEnabled": false
 }
+
+var STATUS_NODE_API_CONFIG* = %* {
+  "APIModules": "provider",
+  "HTTPEnabled": true,
+  "HTTPHost": "0.0.0.0",
+  "HTTPPort": 8545,
+  "WSEnabled": false,
```

Comment:
@kounkou i suggest to disable websocket by default until we start to use it

Reply:
Apologies, I may have overlooked the fact that we decided to stop at websocket. I thought we wanted to test on http and when we hit an update we could easily enable it. Never mind, I've updated the PR just now, WS should work:
```
☁  ~  wscat -c "ws://0.0.0.0:8586"
Connected (press CTRL+C to quit)
> {"jsonrpc": "2.0", "id": 37, "method": "connector_callRPC", "params": ["{\"method\": \"eth_accounts\", \"params\": []}"] }
< {"jsonrpc":"2.0","id":37,"result":"{\"jsonrpc\":\"2.0\",\"id\":0,\"result\":[\"0x6d91b7d498d41a2afb830853f78fdc004b513297\"]}"}
```
I would like to keep the http server running for a while, as it is very convenient to test it with curl.